bigml/api.py

```diff
@@ -186,12 +187,25 @@ def __init__(self, username=None, api_key=None, dev_mode=False,
         `bigml.io` if missing. The expected domain argument is a string or a
         Domain object. See Domain class for details.
 
+        When project is set to a project ID,
+        the user is considered to be working in an
+        organization project. The scope of the API requests will be limited
+        to this project and permissions should be previously given by the
+        organization administrator.
```

Comment:
do you think could we extend this a little bit so that it's possible to be working in a project that doesn't belong to an organization? (i find this weird, but i know is how the api works, so i'm just asking for a workaround): for instance, we could say that if `project=....` and `organization=False`, then one works on the personal project, which doesn't need to belong to an organization... or maybe that already works that way with `organization=None` and we just need to document it?

Reply:
I think if the API allows it, that's what it's going to happen. Let me try and add it to docs if it works.

Reply:
I've changed the code to force using the project in the create/update calls even if it's a user's project